simulator/examples/primitives-fill-macros.rs

```diff
@@ -1,4 +1,6 @@
-//! Demonstrate usage of primitives like `fill.rs` but use macros instead for shorter code
+//! # Example: Using macros to define and style primitives
+//!
+//! Demonstrate usage of primitives like `fill.rs` but use macros instead for shorter code.
```

Comment:
This isn't accurate anymore. `fill.rs` is shorter than this version, because it reuses the same style for multiple primitives. Should we keep the code as is and remove the comment or also reuse the styles in this example?